setup-renv/README.md

```diff
@@ -23,6 +39,15 @@ steps:
     cache-version: 2
 ```
 
+## Quoting R expressions
+
+Several input arguments must be specified as an R expression.
+This increases flexibility, but it also causes some inconvenience, since
+these expressions often need to be quoted in the YAML file.
+A handy tip is is that if your R expression does not contain a single quote,
```

Comment:
```suggestion
A handy tip is that if your R expression does not contain a single quote,
```

Reply:
This is directly copied from https://github.com/r-lib/actions/blob/20ffd99cc10dbdca1582dc5d86a928ec86aa2673/check-r-package/README.md?plain=1#L62 so the change should probably be copied there in another PR.